shapeways/oauth2_client.py

```diff
@@ -13,6 +14,28 @@
 ORDERS_URL = '/orders/v1'
 SINGLE_ORDER_URL = '/orders/{order_id}/v1'
 
+# Relevant headers
+SW_RATE_LIMIT = "SHAPEWAYS"
+CF_RATE_LIMIT = "CLOUDFLARE"
+SW_RATE_LIMIT_LIMIT = 'x-ratelimit-limit'
+SW_RATE_LIMIT_REMAINING = 'x-ratelimit-remaining'
+SW_RATE_LIMIT_RETRY = 'x-ratelimit-retry-inseconds'
+SW_RATE_LIMIT_WINDOW = 'x-ratelimit-window-inseconds'
+CF_RATE_LIMIT_RETRY = 'retry-after'
```

Comment:
let's order these

Reply:
order by what?

Reply:
fi the SW_RATE_LIMIT_RETRY and CF_RATE_LIMIT_RETRY together. but it's not important